### DISCUSSION TITLE
建议把Campaigns改为Hackthon

### DISCUSSION THREAD

**Discussion by LidamaoHub** [BeWaterXYZ/bewater-frontend#565] [General]
同时建议直接就可以看当前的黑客松列表,而不是必须登录后才可以看到黑客松列表

**Comment by leeduckgo** [comment]
> 同时建议直接就可以看当前的黑客松列表,而不是必须登录后才可以看到黑客松列表

Done for the 1st one, and doing the 2nd one, 🤘🏻.
